catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -178,7 +178,8 @@ function ExposeLinkedData({ bucketCfg, bucket, name, revision }) {
 function PkgCode({ data, bucket, name, revision, path }) {
   const code = data.case({
     Ok: ({ hash }) => {
-      const nameWithPath = path ? `${name}/${path}` : name
+      const pathPy = path ? `, path="${path}"` : ''
+      const pathCli = path ? ` --path="${path}"` : ''
```

Comment:
If we use camelcase consistently even for acronyms then should name `parseJSON` as `parseJson`, `ReactDOM` as `ReactDom`, and so on. This style is used by Google, for example

But I think `pathCLI` is more consistent with our code

Reply:
yep, i agree, and i try to keep it consistent, but some things (specifically `*JSON` and `ReactDOM`) are just too deeply ingrained